packages/backend/src/models/SchedulerModel/index.ts

```diff
@@ -140,7 +140,15 @@ export class SchedulerModel {
     }
 
     async getAllSchedulers(): Promise<SchedulerAndTargets[]> {
-        const schedulers = this.database(SchedulerTableName).select();
+        const schedulers = this.database(SchedulerTableName)
```

Comment:
Avoid running scheduled deliveries from deactivated users. 
We should inform admins about this when deactivating users.